test/conformance/go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/ovn-org/ovn-kubernetes/test/conformance
 
-go 1.19
+go 1.20
```

Comment:
I assume this "conformance" dir is only for ANP?

Reply:
correct! and yes good catch there with the go bump.. dangit
TODO: Surya to get back to this.